backend/services/issue/patchIssues.js

```diff
@@ -14,8 +16,8 @@ module.exports = async (req, res) => {
         },
       },
     );
-    res.json();
+    return res.json();
   } catch (err) {
-    res.sendStatus(500);
+    return res.sendStatus(500);
```

Comment:
return 을 추가하신 이유가 있나요?
res.json()에는 res.end가 포함되어서 굳이 return이 필요하지 않을것 같습니다!

Reply:
앞에 if문에서 return으로 조건들을 거르고 있는데 앞에만 적용하니 lint 줄이 떠서 추가해줬습니다ㅠㅠ return 할거면 다 하고 말거면 다 말아라 하더군요...

Reply:
```suggestion
  if (!Array.isArray(issuesId) || typeof isClose !== 'boolean'){
     `const err = new Error('양식이 맞지 않습니다.');
      err.type = 'typeError';
      throw err;`
      } return res.json();
  }catch (e) {
    if (e.type === 'typeError') {
      res.sendStatus(400);
    } else {
      res.sendStatus(500);
    }
  }
```
이런식으로 에러를 던져주는건 어떨까요?

Reply:
반영했습니다ㅎㅎ